api/game/index.js

```diff
@@ -0,0 +1,40 @@
+/* eslint-disable no-underscore-dangle */
+import EventEmitter from 'events';
+import R from 'ramda';
+
+export const READY = 'ready';
+export const WAITING = 'waiting';
+
+export default class Game extends EventEmitter {
+  constructor(id, player) {
+    super();
+    this.id = id;
+    this.status = WAITING;
+    this.players = {
+      [player.username]: player,
+    };
+    this.board = new Array(9).fill(null);
+  }
+
+  addPlayer(newPlayer) {
```

Comment:
it's not a boolean method, should throw an exception in case user already belongs to game